corehq/pillows/utils.py

```diff
@@ -89,3 +93,17 @@ def get_all_expected_es_indices():
 def format_form_meta_for_es(form_metadata):
     form_metadata['appVersion'] = form_metadata['appVersion'].get('#text')
     return form_metadata
+
+
+def update_mapping_migration_op(index, quiet=True):
```

Comment:
[jinx](https://github.com/dimagi/commcare-hq/pull/31402/files#diff-a57560eee274245d02ddc59a1bba68a8830b8b52d5068cd657f2a073f4f2176dR183)

Reply:
I'm going to refactor this. I like the location of yours better.